### PR TITLE
doc/md/cli: intro page

### PR DESCRIPTION
**Pull request by yonidavidson**
image

image

### CODE REVIEW

File: doc/md/CLI/welcome.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Move the atlas binary to a file location on your system PATH.
+
+Example for Mac:
+```shell
+sudo mv ./atlas-darwin-{arch}-{version} /usr/local/bin/atlas
+sudo chown root: /usr/local/bin/atlas
+```
+Example for Linux:
+```shell
+sudo install -o root -g root -m 0755 ./atlas-linux-{arch}-{version} /usr/local/bin/atlas
```
Comment (masseelch) [inline doc/md/CLI/welcome.md:32]: Curious (I don't know much about atlas yet), why are sudo right needed?

Comment (yonidavidson) [reply doc/md/CLI/welcome.md:32]: just for installing in that folder, isn't that always?

Comment (masseelch) [reply doc/md/CLI/welcome.md:32]: And why are you not using `go install`? It makes the executable available as well, doesn't it?

Comment (yonidavidson) [reply doc/md/CLI/welcome.md:32]: I don't want this CLI to be "affiliated" with gophers but rather anyone who uses databases.

File: doc/md/CLI/welcome.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+id: Welcome to Atlas
```
Comment (rotemtam) [inline doc/md/CLI/welcome.md:2]: atlas is the project, which already has an intro page,i think this page presents the command line getting started instructions?

File: doc/md/CLI/welcome.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+slug: /CLI/
+sidebar_position: 2
+---
+
+## CLI
+
+Work with any data source from the command line.
```
Comment (rotemtam) [inline doc/md/CLI/welcome.md:9]: For now at least, this is an incorrect claim, and i think we chose "a database toolkit" as the tag line for now until we come up with something better.

File: doc/md/CLI/welcome.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+![Alt Text](https://release.ariga.io/images/assets/atlas-intro.gif)
```
Comment (rotemtam) [inline doc/md/CLI/welcome.md:11]: Alt Text?

File: doc/md/CLI/welcome.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Installation
+
+Download from one of our links or using curl.
+```shell
+curl -LO https://release.ariga.io/atlas/atlas-{os}-{arch}-{version}
```
Comment (rotemtam) [inline doc/md/CLI/welcome.md:17]: give a copy-able instructions for each platform. 
reduce friction for the user.

File: doc/md/CLI/welcome.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+```shell
+sudo install -o root -g root -m 0755 ./atlas-linux-{arch}-{version} /usr/local/bin/atlas
+```
+
+|Latest Release                        |
+|--------------------------------|
+| [atlas-darwin-amd64-v0.0.1](https://release.ariga.io/atlas/atlas-darwin-amd64-v0.0.1)            |
+| [atlas-darwin-arm64-v0.0.1](https://release.ariga.io/atlas/atlas-darwin-arm64-v0.0.1)          |
+| [atlas-linux-amd64-v0.0.1](https://release.ariga.io/atlas/atlas-linux-amd64-v0.0.1)          |
+| [atlas-windows-amd64-v0.0.1.exe](https://release.ariga.io/atlas/atlas-windows-amd64-v0.0.1.exe)          |
```
Comment (rotemtam) [inline doc/md/CLI/welcome.md:40]: It's cool to a table of all releases and aggregate them here. But add columns "version", "platform", link"
you can have the latest version be "latest (v0.0.1)".

File: doc/md/CLI/welcome.md (hunk 2 above)
Comment (yonidavidson) [reply doc/md/CLI/welcome.md:2]: 👍

File: doc/md/CLI/welcome.md (hunk 3 above)
Comment (yonidavidson) [reply doc/md/CLI/welcome.md:9]: Removed, will update the atlas docs.

File: doc/md/CLI/welcome.md (hunk 4 above)
Comment (yonidavidson) [reply doc/md/CLI/welcome.md:11]: 👍

File: doc/md/CLI/welcome.md (hunk 6 above)
Comment (yonidavidson) [reply doc/md/CLI/welcome.md]: I am taking this out, in the near future we need to think about managing releases (usually it's a page in github).

**Review by rotemtam — APPROVED** [review]

**Pull request merged**